### DISCUSSION TITLE
Proper use of client with ACLs?

### DISCUSSION THREAD

**Discussion by tdcockers** [abraunegg/onedrive#2377] [Q&A] [answered]
I'm using the client to sync files from a Nethserver (Centos) server to Onedrive. Each folder/library is owned by root but has a unique group on it that sets who can access what on the local network.

I notice the file/directory permissions options in the config file... is the correct way to download files from Onedrive and apply the correct permission using the config file or should it be done at system level with sticky bits or something like that?

**Comment by abraunegg** [comment, marked as answer]
@tdcockers 
> I notice the file/directory permissions options in the config file... is the correct way to download files from Onedrive and apply the correct permission using the config file or should it be done at system level with sticky bits or something like that?

As per the documentation: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#sync_dir-directory-and-file-permissions

Special permission bits (setuid, setgid, sticky bit) are not supported.

![image](https://user-images.githubusercontent.com/4956234/231886655-9bb514ba-5300-404b-8abd-4184ea9cbd6e.png)

Your only option here is to correctly set the root folder (the 'sync_dir') user and group, then use the application permission control to set the files & folders correctly.

Users that need to access the data must be part of the 'group' that the 'sync_dir' has been set with.

**Comment by tdcockers** [comment]
Sorry I missed the docs looking on my phone :(

Working well now as per your recommendation thanks.
